kalite/distributed/static/js/distributed/topics/views.js

```diff
@@ -104,28 +115,33 @@ window.SidebarView = BaseView.extend({
     },
 
     resize_sidebar: function() {
-        if ($(window).width() < 1260)
-            this.resize_for_narrow();
+        if (this.state_model.get("open")) {
+            if ($(window).width() < 1260)
+                this.resize_for_narrow();
```

Comment:
Noooo! Use the curly brackets! I don't know why this passed the linter. (If you truly must omit curly brackets, then at least don't put the statement to be executed on a new line... it suggests that more lines could be added to this `if` block, but really only the first statement will be conditionally executed and all subsequent ones will be unconditionally executed).